content/reference/pkg/python/pulumi_datadog/gcp/_index.md

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
Have we shipped Datadog publicly yet?

Reply:
Datadog package has been published for a few days so I've been asking to get the repo opened to allow this